plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -165,6 +165,7 @@ func prioritizeNodes(pod *api.Pod, podLister algorithm.PodLister, priorityConfig
 		}
 	}
 	for host, score := range combinedScores {
+		glog.V(10).Infof("Host %s Score %d", host, score)
```

Comment:
I think we started V(10)'ing because we found that clients with large clusters given us huge logs filled with too much noise, since this happens for each pod for each node.